lib/surveyor/engine.rb

```diff
@@ -1,8 +1,6 @@
 require 'rails'
 require 'surveyor'
 require 'simple_form'
-require 'bootstrap-sass'
-require 'haml' # required for view resolution
```

Comment:
We don't sue this engine at all right? Can we just remove it all? The only part that seems valuable is the factories stuff.

Reply:
I think I tried removing it and it broke in our app- I can poke at it more though.

Reply:
Yeah when I removed the `require 'surveyor/engine' if defined?(Rails) && Rails::VERSION::MAJOR >= 3` from surveyor.rb I kept getting uninitialized constant errors

Reply:
Interesting! I didn't even realize that was there... In that case ignore me.